### PR TITLE
Updates to authentication protocol

### PR DESCRIPTION
**Pull request by MattNguyen**

### CODE REVIEW

File: protocols/authentication.asciidoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+The details about signing procedure can be found below.
+
+==== Obtaining API keys and secrets
+
+// TODO - add step by step instructions
+API keys can be obtained here <link>.
+
+==== Generating credentials_secret
+
+To initiate a session after establishing a WSS connection to a service, users must send a <<login>> message to the backend using their email as the `credentials` and a generated `credentials_secret`.
```
Comment (kernelbee) [inline protocols/authentication.asciidoc]: @MattNguyen : I found the following style is more suitable, please consider it too:
```
<<login>> => <<login, login>>
```

File: protocols/authentication.asciidoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+The `credentials_secret` is a specially constructed JSON Web Token (JWT) that is signed using the user's API `secret_key`. The following JWT claims are required in order to use submit a valid `credentials_secret`:
+
+- `client` - Set to `api`.
+- `uri` - The production endpoint of the requested service.
+- `nonce` - Monotonically increasing number. Usually, a timestamp can be used.
```
Comment (kernelbee) [inline protocols/authentication.asciidoc]:
```
increasing number => increasing integer number 
```
?

File: protocols/authentication.asciidoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- `iat` - The time at which the JWT was issued, in seconds since Epoch.
```
Comment (kernelbee) [inline protocols/authentication.asciidoc]:
```
The <<https://en.wikipedia.org/wiki/Unix_time, Unit time>> at which the JWT was issued, in seconds.
```
?

File: protocols/authentication.asciidoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- `exp` - The expiration time on and after which the JWT must not be accepted for processing, in seconds since Epoch. Must be less than iat+30sec.
```
Comment (kernelbee) [inline protocols/authentication.asciidoc]: The expiration **Unix** time on and after ...

**Review by kernelbee — COMMENTED** [review]
@MattNguyen : if you have time, please apply the suggested and let's merge then. Otherwise I will do it - let me know

**Comment by MattNguyen** [comment]
@kernelbee I force-pushed an update with your suggestions.

**Review by kernelbee — APPROVED** [review]
THANK YOU MATT

**Pull request merged**